### DISCUSSION TITLE
Please help - spawn EPERM

### DISCUSSION THREAD

**Discussion by mikaraduan** [aclap-dev/video-downloadhelper#402] [Help and Q&A]
### Your question

This error has appeared even with the license installed. I do not know what to do. I need help. Please!
Grateful.

spawn EPERM
moz-extension://fd61081d-7a86-4238-82cc-baecae01967b/background/main.js:1
receive@moz-extension://fd61081d-7a86-4238-82cc-baecae01967b/background/main.js:1:13157
nd</callCatchAppNotFound/</<@moz-extension://fd61081d-7a86-4238-82cc-baecae01967b/background/main.js:1:42626 

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: pt-BR
license: accepted
key (only 16 first characters): 18c193e0e7d44489
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\mikar",
  "converterError": "spawn EPERM"
}

**Comment by jcfrog** [comment]
Hi,
Did you change the download directory?

**Reply by mikaraduan** [reply]
Hi, thanks for helping. But I believe not. I'm a very basic user.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/167039257/87ac5e01-5258-482b-ac0e-41087e40a093)


And I tried to put it down there as a default too. But it still has the same error.

**Reply by mikaraduan** [reply]
I have already uninstalled and installed again and still have the error.

**Reply by mikaraduan** [reply]
Guys, I discovered what was "wrong".
I don't know why, or what it has to do with it, but I had changed the webcam settings in my antivirus. I had blocked everything out of fear.
This was what was causing the error.
How crazy, right?!
But thank you so much for trying to help me.
Warm hugs from Brazil :)

**Comment by GrampaWildWilly** [comment]
You might want to look at #399.

**Reply by mikaraduan** [reply]
Hi, thanks for helping me.
I'm a very basic user. But on every site I try to download the same error occurs.

**Reply by mikaraduan** [reply]
Guys, I discovered what was "wrong".
I don't know why or what it has to do with it, but I had changed the webcam settings in my antivirus. I had blocked everything out of fear.
This was what was causing the error.
How crazy, right?!
But thank you so much for trying to help me.
Warm hugs from Brazil :)
